### PR TITLE
First try

### PR DESCRIPTION
**Pull request by Mar-Alex**

### CODE REVIEW

File: students [hunk 1]
```diff
@@ -5,3 +5,4 @@ Tatyana Korchinova - https://github.com/tetyanko/myfirstproject
 Irina Yeremenko - https://github.com/IrinaYeremenko/HomeWork_0
 Diana Sinenchenko - https://github.com/dianasinenchenko/example0
 Lera Kulikova - https://github.com/lera777/HomeWork0
+Oleksandr Marukhnenko - https://github.com/Mar-Alex/environment.hw1
```
Comment (xSAVIKx) [inline students]: You have to add URL to your own first repository, but not to your fork.

**Review by xSAVIKx — CHANGES_REQUESTED** [review]
@Mar-Alex please fix your PR and add some meaningful description to the PR.

**Comment by Mar-Alex** [comment]
Correct link was added

**Comment by xSAVIKx** [comment]
@Mar-Alex LGTM

**Pull request closed without merging**